addons/mail/models/mail_thread.py

```diff
@@ -2214,6 +2214,10 @@ def _notify_record_by_inbox(self, message, recipients_data, msg_vals=False, **kw
         if bus_notifications:
             self.env['bus.bus'].sudo().sendmany(bus_notifications)
 
+    @staticmethod
+    def _sanitize_email_subject(subject):
```

Comment:
Well I know I asked you to make a method, but actually it looks weird ... think it is better to simply remove it, it actually adds more noise than helps anyone. Sorry for back and forth ^^ . A small comment above the inlined line would be better actually.